src/styles/index.scss

```diff
@@ -1,3 +1,7 @@
+@import 'src/styles/utils/variables';
```

Comment:
Ensure that the file `src/styles/utils/variables.scss` exists and contains the necessary variable definitions used in your other SCSS files.